wasm-engines.md

```diff
@@ -0,0 +1,208 @@
+# Wasm Engines
+
+[Binaryen](https://github.com/WebAssembly/binaryen.git) and
+[Wabt](https://github.com/WebAssembly/wabt.git) are two [official WebAssembly
+Projects](https://github.com/WebAssembly/) providing a set of tools to interact
+with WebAssembly.
+
+Wabt's focus is on the manipulation of WebAssembly binary files (`wasm`) and
+text format (`wast`) and conversion between the two formats. Binaryen provides a
+compiler and toolchain infrastructure library for WebAssembly. It aims to make
+compiling to WebAssembly easy, provides a C API, internal IR, and optimizer.
+Similar to Wabt, Binaryen also provides a set of tools to interact with
+WebAssembly binary files and WebAssembly in Text Format. The rest of this
+document explains how to use these tools to work with Ewasm contracts.
+
+By default, [Hera](https://github.com/ewasm/hera) ships with binaryen support,
```

Comment:
What is the implication for the person reading this doc? Why do they need to care which engine(s) hera supports? E.g., should we tell them to make sure they're decompiling/recompiling using the same engine as the one they're using with Hera to ensure compatibility?

Reply:
thanks @lrettig , updated the document.